webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -49,7 +49,11 @@ export default class CreateIssueModal extends PureComponent {
     componentDidUpdate(prevProps) {
         if (this.props.post && !prevProps.post) {
             this.setState({issueDescription: this.props.post.message}); //eslint-disable-line react/no-did-update-set-state
-        } else if (this.props.channelId && (this.props.channelId !== prevProps.channelId || this.props.title !== prevProps.title)) {
+        } else if (
+            this.props.channelId &&
+            (this.props.channelId !== prevProps.channelId ||
+                this.props.title !== prevProps.title)
+        ) {
```

Comment:
Can we revert these indentation changes in the PR, since they are unrelated to the UI styling issue being fixed here?